plugin-build/buildSrc/src/main/kotlin/io/sentry/android/gradle/internal/ASMifyTask.kt

```diff
@@ -0,0 +1,88 @@
+package io.sentry.android.gradle.internal
+
+import org.gradle.api.tasks.Exec
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.options.Option
+import org.gradle.api.tasks.options.OptionValues
+import java.io.File
+import java.io.FileOutputStream
+import java.util.*
+
+/**
+ * A task which outputs a asmified version of a given .class file. This is useful, when
+ * the ASM IntelliJ Plugin fails to decompile nested synthetic classes (e.g. lambdas).
```

Comment:
if I got the description correctly, this should not be part of the plugin-build then, right, since its not required to do the bytecode man work.

Reply:
I mean, it's not strictly required, but it's a helper task for bytecode instrumentation, I still used it when developing. I'm open to move it somewhere else, I just don't know what's the better place, since the task is related to bytecode instrumentation, plus, it also uses asm dependencies from the plugin-build classpath

Reply:
got it, the main point is, this is used for developing the plugin and not executing the plugin.
the class is in the `buildSrc` folder tho, so this class isn't packaged by the final jar, is that correct? if so, I am fine with it

Reply:
Yeah, I think that's the point of `buildSrc` folder, to have assisting tasks and plugins in there, that are added to the classpath of the project, but are not being published (https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#sec:build_sources)

Reply:
yep, I just missed this when writing the comment, but 2nd pass made more sense, thanks, resolving the comment.